inference-engine/src/transformations/include/transformations/utils/utils.hpp

```diff
@@ -49,6 +51,17 @@ bool has_op_with_type(const std::shared_ptr<const ngraph::Function> &function) {
     }
     return false;
 }
+
+inline bool has_decompression_converts(const std::shared_ptr<const ngraph::Function>& function) {
+    for (const auto& op : function->get_ops()) {
+        if (std::dynamic_pointer_cast<ngraph::opset8::Convert>(op)) {
+            if (ov::is_decompression(op))
+                return true;
```

Comment:
does not look optional - can we implement BFS / DFS algo and use here? 
In this case we don't have to traverse the whole graph in most cases

Reply:
Why? This algo will end when first Convert with the attribute is met. I don't understand how BFS/DFS will improve that

Reply:
here you take ALL the nodes first, then go over all nodes one more time.
If you use BFS / DFS - you can find node with `ov::is_decompression(op) == true` in general much faster